### PR TITLE
solution

### PR DESCRIPTION
**Pull request by yelnikov-andrii**
- [DEMO LINK](https://yelnikov-andrii.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,19 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 48px;
+  justify-content: center;
 
   max-width: 944px;
```
Comment (rostyslav-pyliak) [inline src/styles/_catalog.scss:7]: You can't set width for BEM block

**Review by rostyslav-pyliak — CHANGES_REQUESTED** [review]
Something wrong with DEMO LINK. Doesn't work

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]
Is your grid working properly on a small screen?

**Comment by yelnikov-andrii** [comment]
> Is your grid working properly on a small screen?

yes, in a one column 
image

**Comment by yelnikov-andrii** [comment]
> 
on 480px screen-width two columns

image

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**